subsys/bluetooth/host/Kconfig

```diff
@@ -249,6 +249,12 @@ config BT_AUTO_PHY_UPDATE
 	  want to rely on remote device to initiate the procedure at its
 	  discretion.
 
+config BT_REMOTE_VERSION
```

Comment:
@jhedberg we should move this to the common Kconfig settings so that I can refer to it in https://github.com/zephyrproject-rtos/zephyr/pull/19657

Reply:
@carlescufi What exactly do you want to do on the controller side? Don't we want this always enabled for split (controller only) builds? AFAIK it's a mandatory command for the controller whenever connections are supported.

Reply:
I meant enabling the processing in the priority thread. But in fact what I'll do is to conditionally compile it based on whether it's a combined build or not, so let's not move it. Sorry for the noise.